libs/external/shadcn/components/ui/date-time-picker.tsx

```
@@ -0,0 +1,554 @@
+        setInputValue(formatDateTime(parsedDateTime))
+        onTimeChange(`${hour}:${parsedDateTime.getMinutes()} ${PM_AM}`)
+      }
+    },
+    [value],
+  )
+
+  const handleKeydown = React.useCallback(
+    (e: React.KeyboardEvent<HTMLInputElement>) => {
+      switch (e.key) {
+        case 'Enter':
+          const parsedDateTime = parseDateTime(e.currentTarget.value)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)